code/io/timer.cpp

```diff
@@ -26,9 +26,10 @@ static int Timer_inited = 0;
 
 
 #define MICROSECONDS_PER_SECOND 1000000
-
 #define NANOSECONDS_PER_SECOND 1000000000
 
+static long double Timer_factor;
```

Comment:
This should be renamed to indicate that this factor should be used for converting nanosecond values.